### PR TITLE
Add ports support to dell show vlan {id}

### PR DESCRIPTION
**Pull request by fbouliane**
Supports vlan with multiple line for ports.
Supports interface using more than one letter (2/xg1).
Supports sequenced interface 1/g1-1/g2

### CODE REVIEW

**Comment by godp1301** [comment]
:+1:

File: fake_switches/dell/command_processor/enabled.py [hunk 1]
```diff
@@ -204,19 +206,53 @@ def show_vlan_page(self, vlans):
...
             self.write("--More-- or (q)uit")
             self.on_keystroke(self.continue_vlan_pages, vlans)
 
+    def get_ports_for_vlan(self, vlan):
+        ports = []
+        for port in self.switch_configuration.ports:
+            if not isinstance(port, VlanPort):
+                if (port.trunk_vlans and vlan.number in port.trunk_vlans) \
+                        or port.access_vlan == vlan.number \
+                        or port.trunk_native_vlan == vlan.number:
```
Comment (lindycoder) [inline fake_switches/dell/command_processor/enabled.py]: Not sure i see a test for this one

**Comment by emmurd** [comment]
:+1:

**Comment by lindycoder** [comment]
:+1:

**Pull request merged**